drivers/openstack/openstack.go

```diff
@@ -318,6 +319,35 @@ func (d *Driver) GetIP() (string, error) {
 	return "", fmt.Errorf("No IP found for the machine")
 }
 
+func (d *Driver) GetPrivateIP() (string, error) {
```

Comment:
I just test it in the openstack which is using nova network.

docker-machine --debug create \
--driver openstack \
--openstack-auth-url http://172.19.0.2:5000/v2.0 \
--openstack-username demo \
--openstack-password password \
--openstack-tenant-name demo \
--openstack-flavor-id 4 \
--openstack-image-id 312a1b0c-e0b2-4968-a828-3e0b67c842d9 \
--openstack-ssh-user centos \
--openstack-sec-groups default \
--openstack-floatingip-pool public \
--openstack-nova-network=true \
openstack02